website_sale_hide_empty_category/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Website Sale - Hide Empty Categories",
+    "summary": "Hide any Product Categories that are empty",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://laslabs.com/",
```

Comment:
@remytms Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

Reply:
@nikul-serpentcs , Thanks for your review ! :) It's now fixed.